src/services/storage.js

```diff
@@ -17,135 +17,119 @@
     with this program; if not, write to the Free Software Foundation, Inc.,
     51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 */
-import PouchDB from 'pouchdb-react-native';
+import AsyncStorage from '@react-native-community/async-storage';
 import merge from 'lodash.merge';
 import { Sentry } from 'react-native-sentry';
+import PouchDBStorage from './pouchDBStorage';
 
-function Storage(name: string, opts: ?Object = {}) {
+const STORAGE_SETTINGS_KEY = 'storageSettings';
+
+function Storage(name: string) {
   this.name = name;
-  this.opts = { ...opts };
-  this.connect();
+  this.prefix = `wallet-storage:${this.name}:`;
 }
 
-Storage.prototype.connect = function () {
-  this.needToReconnect = false;
-  this.connection = new PouchDB(this.name, this.opts);
-  console.log(`Connected to the database ${this.name}`);
-};
-
-Storage.prototype.db = function () {
-  if (this.needToReconnect) this.connect();
-  return this.connection;
+Storage.prototype.getKey = function (id: string) {
+  return this.prefix + id;
 };
 
-Storage.prototype.get = function (id: string) {
-  return this.db().get(id).catch(() => ({}));
+Storage.prototype.get = async function (id: string) {
+  const data = await AsyncStorage.getItem(this.getKey(id))
+    .then(JSON.parse)
+    .catch(() => {});
+  return data || {};
```

Comment:
Just curious, wouldn't it make sense to return `null`/`undefined` when the key is not found?

Reply:
The main idea is to keep this storage wrapper backwards-compatible with the previous one